core/drivers/crypto/hisilicon/include/hpre_dh.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2022, HiSilicon Technologies Co., Ltd.
```

Comment:
`2022-2024`

Reply:
![image](https://github.com/OP-TEE/optee_os/assets/118411745/55655e4c-8c13-4a6b-9a80-25041957a66d)
drvcrypt.h and initcall.h is needed in dh.c. I think these head files is already included, such as trace.h is included in drvcrypt.h

Reply:
The general rule is to include all the header files you are using. For instance if you use traces then you include <trace.h> even if it is included by another header. The reason is because relying on indirect inclusions can break if the header files change.

Reply:
Get it.  Solved.